app/views/organizations/new_account.html.slim

```diff
@@ -0,0 +1,31 @@
+.page-header
+  h2 New organization bank account
+
+.row.previous_buttons
+  .col-sm-6
+    p
+      ' Bank account will be created by default (with default currency)
+      br
+      br
+      = link_to 'Create default Bank account', default_account_organization_path(@organization.id), class: 'btn btn-primary'
```

Comment:
@dissident If I understand right this leads to some action, so it should be button, not link.